### PR TITLE
Document how to perform a release

### PR DESCRIPTION
**Pull request by hatched**
## Done

Document how to perform a release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,3 +271,63 @@ const wrapper = mount(
...
+
+- Take this tarball and create a new release on GitHub under the appropriate version tag.
+- Add the notes from the changelog to the release notes on GitHub.
+
+### Release to jaas.ai
+
+The release to jaas.ai is in two steps. New releases of the jaas.ai website will pull in the
+latest release of the Dashboard for JAAS.
+
+- Visit the CI instance and release a new version of jaas.ai to staging.
```
Comment (anthonydillon) [inline README.md:317]: Can you add links here please

File: README.md [hunk 2]
```diff
@@ -244,3 +271,63 @@ const wrapper = mount(
...
+- Validate that the website works as expected and that the [release tests](#release-tests) are pass successfully.
+
+### Release to Juju Simplestreams
+
+When Juju bootstraps a new instance it fetches the appropriate content for the infrastructure it's deployed on from a system called Simplestreams.
+
+To release a tarball of the Dashboard to Simplestreams:
+
+- Clone the release script [Shipit](https://github.com/CanonicalLtd/shipit).
+- Follow the instructions in that projects `README.md` to perform the release. This will take aproximately an hour but it's automated. You'll want to keep an eye on it as it's doing its thing in the event it needs input.
```
Comment (anthonydillon) [inline README.md:329]: Do you need access our permission to sign the gpg key and push?

File: README.md [hunk 3]
```diff
@@ -244,3 +271,63 @@ const wrapper = mount(
...
+### Release prep
+
+To prepare to release:
+
+- Clone a new copy of the repository to remove the chance of any uncommitted artifacts ending up in the release.
+- Thoroughly [QA the Dashboard](#qa'ing-the-dashboard) following the [Release tests](#release-tests)
+- Update the changelog by adding the important commits from the last tag.
+
+```
+  git log `git describe --tags --abbrev=0`..HEAD --format='- %b' --merges | sed '/^- $/d'
```
Comment (anthonydillon) [inline README.md:294]: Did you look into the system maas-ui uses?

File: README.md [hunk 4]
```diff
@@ -244,3 +271,63 @@ const wrapper = mount(
...
+- [Release to jaas.ai](#release-to-jaas.ai)
+- [Release to Juju Simplestreams](#release-to-juju-simplestreams)
+- [Create an announcement on the Juju Discourse](#create-an-announcement-on-the-juju-discourse)
+
+### Release prep
+
+To prepare to release:
+
+- Clone a new copy of the repository to remove the chance of any uncommitted artifacts ending up in the release.
+- Thoroughly [QA the Dashboard](#qa'ing-the-dashboard) following the [Release tests](#release-tests)
```
Comment (anthonydillon) [inline README.md:290]: This is a big ask on s lightweight release. They should of course QA it through.

File: README.md (hunk 2 above)
Comment (hatched) [reply README.md:329]: With the latest update to `shipit` it no longer requires that, but you have to be in the appropriate groups on launchpad.

File: README.md (hunk 3 above)
Comment (hatched) [reply README.md:294]: I did and it looks pretty good, there are a few others I've tried to but as far as the scope for this document I just included a simple snippet. We can certainly iterate on this process over time.

File: README.md (hunk 4 above)
Comment (hatched) [reply README.md:290]: Yeah I feel like over time we'll be able to enhance our automated testing infrastructure and these steps (which I haven't actually written yet) will be dramatically reduced. But the issue right now is that none of that exists, and these releases will go out into peoples production environments so we want to make sure they are thoroughly qa'd. So I'm err'ing on the side of caution in this document around these steps.

File: README.md (hunk 1 above)
Comment (hatched) [reply README.md:317]: good catch, done

**Pull request merged**